src/coordinator/roles.js

```diff
@@ -46,6 +64,93 @@ class Roles {
     this._sdk = sdk;
   }
 
+  /**
+   * Add an application to a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: POST
+   *
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {number} applicationId The ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtRoleApplication}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles
+   *   .addApplication('36b8421a-cc4a-4204-b839-1397374fb16b', '58')
```

Comment:
I think `58` here should be a number rather than a string since you specify it as a number on line 74.

Realistically, it shouldn't really matter since it's getting turned into a string anyways for the request url but just to be consistent with what's listed as a `@param`. What are your thoughts?